README.md

```diff
@@ -238,14 +244,32 @@ The top level of the MxJdf document contains the following elements:
  }
 ```
 
-| Property  | Description | Values |
+| Property | Description | Values |
 |---|---|---|
 |additionalProjectName||Additional Order number. Reference number. (String)||
 |completionType|Urgency of the job. Default value is 4 (Standard)|INVALID(0), <br>SAME_DAY(1), <br>EXPRESS(2), <br>OVERNIGHT(3), <br>STANDARD(4), <br>SAVER(5);|
 |totalCirculation|The total number of copies (Integer)||
 |assumedPrintType|The expect print technology|NONE(0), <br>LITHO(1), <br>DIGITAL(2), <br>INKJET(3)|
+|jobType|Classification of print job|PRIMARY_PRINT(0), <br>RIPPED_PROOF(1), <br>HARD_COPY_PROOF(2)|
 
-# 12. Element: shipment
+# 12. Element: associatedProof
+```javascript
+"associatedProof": {
+    "proofJobNumber": "532809",
+    "proofJobType": 1,
+    "proofDate": 1579177108174
```

Comment:
Why not human readable in standardised format?

Reply:
Very good question.

I used this approach to dates to match what's already there for `dateCreated` (top-level object).